app/services/healthcheck.rb

```diff
@@ -1,30 +1,22 @@
-class Healthcheck
+module Healthcheck
```

Comment:
Feels like caching an implimentation detail that the controller doesn't need to know about?

If we leave it as `Healthcheck` and happen to take out caching later, then we won't need to rename the module or change the controller code.